netstandard/tools/ConflictItem.cs

```diff
@@ -0,0 +1,178 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using System;
+using System.IO;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    internal enum ConflictItemType
+    {
+        Reference,
+        CopyLocal,
+        Platform
+    }
+
+    // Wraps an ITask item and adds lazy evaluated properties used by Conflict resolution.
+    internal class ConflictItem
+    {
+        public ConflictItem(ITaskItem originalItem, ConflictItemType itemType)
+        {
+            OriginalItem = originalItem;
+            ItemType = itemType;
+        }
+
+        public ConflictItem(string fileName, string packageId, Version assemblyVersion, Version fileVersion)
+        {
+            FileName = fileName;
+            SourcePath = fileName;
+            PackageId = packageId;
+            AssemblyVersion = assemblyVersion;
```

Comment:
Yep, I missed a couple of these cases when plumbing through the new constructor that directly sets the backing fields.  Fixed.